cmd/openshift-install/imagebased.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newImageBasedCmd(ctx context.Context) *cobra.Command {
+	imagebasedCmd := &cobra.Command{
+		Use:   "image-based",
```

Comment:
Until it's no longer just a skeleton?

```suggestion
		Use:   "image-based",
		Hidden: true
```

Reply:
(not necessary if the PR is urgent, just a suggestion)